sotodlib/coords/__init__.py

```diff
@@ -4,3 +4,4 @@
     Timer, DEG, ScalarLastQuat
 )
 from . import planets
+from .plotting import *
```

Comment:
I tend to worry about any code that automatically imports anything from matplotlib (yes, I'm a HUGE fan of pysmurf, /s). There's nothing in there that will crash things if you don't have X11 forwarding set up or anything?